### ISSUE TITLE
Infinite Claims, Increase Price

### ISSUE THREAD

**Issue by cjburkey01** [cjburkey01/ClaimChunk#20]
* Increase price of new chunks according to multiplier.
* Allow infinite chunks (set max to -1)

**Comment by starcrasher87** [comment]
This is a great idea, to deter players from claiming enormous areas, but still give the opportunity to work for the privilege to do so.

**Comment by cjburkey01** [comment]
I suppose I should finally implement this, huh?

**Comment by EggLord54** [comment]
A multiplier sounds like a really cool idea, but i feel like allowing the multiplier to dynamically increase also might be useful, so that it doesn't end up multiplying to infinity...

**Comment by CoderCrazy1** [comment]
Price multiplication would be very useful for my server. Are there still plans to add this?

**Comment by cjburkey01** [comment]
Absolutely, I have a bunch of things planned for 0.1.0 and this'll be one of those things. 0.0.21 is expected to drop after we test it a bit on 1.16, so probably next week some time. After that, I'll be working on 0.1.0 and fix bugs for 0.0.22 as I go along.

**Comment by JustDoom** [comment]
nice

**Comment by cjburkey01** [comment]
What's also nice is how well it demonstrates my work ethic: ![Image showing that this issue is almost 4 years old](https://user-images.githubusercontent.com/9255643/111966251-1595ce00-8acd-11eb-815e-ad2c7eda5f3d.jpeg)
